bin/stage

```diff
@@ -33,15 +32,36 @@
 # Configuration
 # ========================================
 
+# Load in configuration file, or ask and populate.
+def read_or_ask_for_configuration
+  require 'yaml'
+  conf_file = File.join(ENV['HOME'], '.jekyll-stagingrc')
```

Comment:
This puts the configuration here, in `~/.jekyll-stagingrc`. I worry that if I have more than one site using this script, it shall not work. What do you think about using the current working directory such that `stage` would be configurable for each site? This means you may only run stage from the same directory as a configuration file.